haystack/retriever/dense.py

```diff
@@ -229,7 +229,7 @@ def train(self,
               batch_size: int = 2,
               embed_title: bool = True,
               num_hard_negatives: int = 1,
-              num_negatives: int = 0,
+              num_positives: int = 0,
```

Comment:
what about this ?

Reply:
yep, same here. 1 is more meaningful

Reply:
well i think may be there's a bug here, I'm trying this code. 
with batch_size=2
it gave me an error that a vector size is (2) -means query- and the other is (4) -means passages-.

I will change it for the moment, and get back when I'm sure about the bug.

Reply:
Was there a bug or are we ready to merge here?

Reply:
it's all good, (y)